osu.Framework/Input/Events/FocusEvent.cs

```diff
@@ -1,18 +1,27 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using JetBrains.Annotations;
+using osu.Framework.Graphics;
 using osu.Framework.Input.States;
 
 namespace osu.Framework.Input.Events
 {
     /// <summary>
-    /// An event represeting that a drawable gained the focus.
+    /// An event representing that a drawable gained the focus.
     /// </summary>
     public class FocusEvent : UIEvent
     {
-        public FocusEvent(InputState state)
+        /// <summary>
+        /// The <see cref="Drawable"/> that has lost focus, or <c>null</c> if nothing was previously focused.
+        /// </summary>
+        [CanBeNull]
+        public readonly Drawable PreviousFocus;
```

Comment:
```suggestion
        public readonly Drawable PreviouslyFocused;
```

Think this reads better.

Reply:
I tend to agree, and would extend this to renaming `FocusLostEvent.NextFocus -> FocusLostEvent.NextFocused` for parity.